src/main/java/org/ladysnake/blabber/Blabber.java

```diff
@@ -152,6 +153,7 @@ public void onInitialize() {
 		BlabberRegistrar.init();
 		CommandRegistrationCallback.EVENT.register((dispatcher, registryAccess, environment) -> BlabberCommand.register(dispatcher));
 		registerAction(id("command"), CommandDialogueAction.CODEC);
+		registerIllustration(id("list"), DialogueIllustrationCollection.TYPE);
```

Comment:
Personally I like the name `composition` for a collection of illustrations, but otherwise I think most image editing software would call this a `group`

Reply:
group is shorter so group it is!